doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml

```diff
@@ -481,6 +481,25 @@ This property corresponds to the "FailoverPartnerSPN" and "Failover Partner SPN"
             </format>
           </remarks>
         </FailoverPartnerSPN>
+      <HostNameInCertificate>
+        <summary>Gets or sets the host name to use when validating the server certificate for the connection. When not specified, the server name from the `Data Source` is used for certificate validation. (Only available in v5.0+)</summary>
+        <value>
+          The value of the <see cref="P:Microsoft.Data.SqlClient.SqlConnectionStringBuilder.HostNameInCertificate" /> property, or <see langword="String.Empty" /> if none has been supplied.
+        </value>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[  
+
+## Remarks  
+This property corresponds to the "HostNameInCertificiate" and "Host Name in Certificiate" keys within the connection string. 
+
+> [!NOTE]
+> This property only applies when using `Encrypt` in <xref:Microsoft.Data.SqlClient.SqlConnectionEncryptOption.Strict%2A> mode, otherwise it is ignored.
```

Comment:
HNIC should apply in Mandatory or Strict mode. How does the implementation behave?

Reply:
You're right.  I misinterpreted this feature and just verified that mandatory with the HNIC works and doesn't require the `TrustServerCertificate=true` to connect. Thanks!

Reply:
Also verified the same for the `ServerCertificate`